fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -67,11 +68,62 @@ internal class HttpClient(private val proxy: Proxy? = null) : Client {
         }
     }
 
+    override suspend fun awaitRequest(request: Request): Response = suspendCoroutine { continuation ->
+        try {
```

Comment:
Since this implementation is basically the same with `executeRequest` counterpart. Shall we abstract them out so we have a better maintainability later, even better if we can call the `executeRequest` directly?

``` Kotlin
override suspend fun awaitRequest(request: Request): Response = suspendCoroutine {
  try {
    continuation.resume(executeRequest(request))
  } catch(exception: Exception) {
    continuation.resumeWithException(exception)
  }
}

```

Reply:
yes, sorry I should have seen that first time round

Reply:
I have made this change but for some reason it is not showing